src/platforms/dart/config.yml

```diff
@@ -0,0 +1,4 @@
+title: Dart
+caseStyle: canonical
```

Comment:
Options are showing as `before-breadcrumb` instead of camel case: `beforeBreadcrumb`

I guess this is the right format:
```suggestion
caseStyle: camelCase
```

Reply:
yeah but Java is `canonical` and I copy-paste the config, I wondered this too

Reply:
fixed java/android too